lighthouse-core/report/html/renderer/report-renderer.js

```diff
@@ -242,38 +223,33 @@ class ReportRenderer {
       wrapper.appendChild(renderer.render(category, report.categoryGroups));
     }
 
+    const reportFragment = this._dom.createFragment();
+    const topbarDocumentFragment = this._renderReportTopbar(report);
+    reportFragment.appendChild(topbarDocumentFragment);
+
     if (scoreHeader) {
-      const scoreGauges =
-        this._renderScoreGauges(report, categoryRenderer, specificCategoryRenderers);
-      scoreHeader.append(...scoreGauges);
       const scoreScale = this._dom.cloneTemplate('#tmpl-lh-scorescale', this._templateContext);
       const scoresContainer = this._dom.find('.lh-scores-container', headerContainer);
+      scoreHeader.append(
+        ...this._renderScoreGauges(report, categoryRenderer, specificCategoryRenderers));
       scoresContainer.appendChild(scoreHeader);
       scoresContainer.appendChild(scoreScale);
-    }
-
-    reportSection.appendChild(this._renderReportFooter(report));
-
-    const reportFragment = this._dom.createFragment();
 
-    if (!this._dom.isDevTools()) {
-      const topbarDocumentFragment = this._renderReportTopbar(report);
-      reportFragment.appendChild(topbarDocumentFragment);
-    }
-
-    if (scoreHeader && !this._dom.isDevTools()) {
       const stickyHeader = this._dom.createElement('div', 'lh-sticky-header');
-      this._dom.createChildOf(stickyHeader, 'div', 'lh-highlighter');
-
-      const scoreGauges =
-        this._renderScoreGauges(report, categoryRenderer, specificCategoryRenderers);
-      stickyHeader.append(...scoreGauges);
-
+      stickyHeader.append(
+        ...this._renderScoreGauges(report, categoryRenderer, specificCategoryRenderers));
+      // The highlighter is placed in the first gauge to avoid an layout issue where the first
+      // paint comes before the highlighter's calculated position is applied. This causes
+      // the initial positioning to be rendered for a split second. As a workaround, the
+      // highlighter is placed in the first score gauge so that the initial position is correct.
```

Comment:
I think the context of this will be lost over time and not make much sense in isolation (there's no need for a workaround cause the only thing we're doing *is* the workaround). 

Really we just need something like `// Position highlighter at first gauge; will be transformed on scroll.`

(should this div actually be added in `report-ui-features` since it's kind of broken in reports that don't use `report-ui-features` and so don't get the scroll handler?)

Reply:
good call, done